internal/cli/command/root.go

```diff
@@ -42,7 +41,7 @@ func initConfig() {
 	} else {
 		home, err := homedir.Dir()
 		if err != nil {
-			fmt.Println(err)
+			logger.Error("Config Setup Issue : ", err)
```

Comment:
Error strings should not be capitalized. [ref](https://github.com/golang/go/wiki/CodeReviewComments#error-strings)